TypeCobol/Compiler/Diagnostics/Diagnostic.cs

```diff
@@ -113,6 +182,11 @@ protected Diagnostic([NotNull] Diagnostic other)
         public int ColumnEnd { get; private set; }
         public string Message { get; private set; }
         public Exception CaughtException { get; }
+        private object[] _MessageArgs;
+        /// <summary>
+        /// Does this diagnostic use predefined arguments.
```

Comment:
What is a `predefined arguments` ?

Reply:
A predefined argument here stand for a calculated ColumnStart argument value, when the message is relocated after a continuation line relocation, then this predefined argument is detected and a substitution is made to get the real column start value at relocation time.